### PR TITLE
Move reset cache to header

### PR DESCRIPTION
**Pull request by elsiosanchez**
## Bug report / Feature
Move reset cache to header
#### Screenshot or Gif
resetcache
#### Other relevant information
- Your OS: Kubunto
- Web Browser: Chrome
- Node.js version: v 14.0.0
- Yarn version: v 1.22.0
- adempiere-vue version: v 4.4.0

### CODE REVIEW

File: src/layout/components/Navbar.vue [hunk 1]
```diff
@@ -166,6 +170,9 @@ export default {
     isMenuOption() {
       this.isMenuMobile = !this.isMenuMobile
     },
+    isResetCache() {
```
Comment (EdwinBetanc0urt) [inline src/layout/components/Navbar.vue]: change method name to `cacheReset`

File: src/utils/ADempiere/constants/process.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+export const processHeader = {
```
Comment (EdwinBetanc0urt) [inline src/utils/ADempiere/constants/process.js]: export string constant and remove object:
```javascript
export const RESET_CACHE_PROCESS_UUID = 'uuid'
```

**Review by EdwinBetanc0urt — CHANGES_REQUESTED** [review]

File: src/utils/ADempiere/constants/process.js (hunk 2 above)
Comment (elsiosanchez) [reply src/utils/ADempiere/constants/process.js]: resolve

File: src/layout/components/Navbar.vue (hunk 1 above)
Comment (elsiosanchez) [reply src/layout/components/Navbar.vue]: resolve

**Pull request merged**